index.node.js

```diff
@@ -7,8 +7,12 @@ const Oasis = require('./web3c/oasis');
  * Web3c is a wrapper that can be invoked in the same way as Web3.
  * Expects Web3 v1.0
  */
-module.exports = function (provider) {
-  web3.call(this, provider);
+module.exports = function (provider, web3Override, options) {
```

Comment:
is `web3Override` important enough that it should have a positional parameter rather than a key in `options`?

Reply:
Note that we already have a web3 override in `web3c/index.js` (not having it here was an oversight in a previous change).

I think the `web3Override` is indeed important. Nevertheless, my thinking here is that we'd keep `options` to things that affect the `Oasis` namespace.

Note, we already have a web3Override in

Reply:
ok

Reply:
agree that this separate from options makes sense